src/views/CashlinkReceive.vue

```diff
@@ -341,6 +342,7 @@ class CashlinkReceive extends Vue {
             case CashlinkTheme.GENERIC:
             case CashlinkTheme.CHRISTMAS:
             case CashlinkTheme.LUNAR_NEW_YEAR:
+            case CashlinkTheme.EASTER:
                 return 'You are loved';
```

Comment:
maybe "You are loved" for Christmas & Lunar new year. But "Happy easter" or something like that for easter, nah ?